Makefile

```diff
@@ -21,6 +21,9 @@ maintainer-clean: clean
 docs:
 	$(MAKE) -C docs clean man html
 
+update-docs:
+	bash ./docs/update-help-messages.sh
+
```

Comment:
Do I read the script correctly that this target depends on the `install` target?

Reply:
Makes sense. I added the dependency to build target.

Reply:
Another way to do it would be to use a PGCOPYDB variable that points to the binary we're using, and would default to `./src/bin/pgcopydb/pgcopydb` so that the default dependency would be to `make bin` rather than `make install`, or could be completely bypassed with `make PGCOPYDB=/usr/local/bin/pgcopydb` docs` for instance.

That said in any case the pgcopydb binary built from the current repository state needs to be available when running the `update-help-messages.sh` script. Also, I'm not sure that the CI system needs to be able to run that command, as the computed files are checked-in to the repository?

Reply:
I think the CI system should raise an error when the developer updates a command interface and forgets to update the docs. The purpose of said test should not be updating files, but letting the developer know that there is something that needs fixing.